<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QWidget" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Bot</string>
  </property>
  <widget class="QLineEdit" name="userInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>31</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/envoyer.png);
background-color:none;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatDisplay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>textEdit</zorder>
  <zorder>chatDisplay</zorder>
  <zorder>userInput</zorder>
  <zorder>sendButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
